apiserver/facades/client/client/status.go

```diff
@@ -1237,7 +1236,8 @@ func (context *statusContext) processApplication(application *state.Application)
 			sort.Sort(bySinceDescending(versions))
 			processedStatus.WorkloadVersion = versions[0].Message
 		}
-	} else {
+	}
```

Comment:
This isn't right, the above block to get the workload version is now run for all models.

```
...
	planRequired := metrics != nil && metrics.Plan != nil && metrics.Plan.Required
	if planRequired || len(application.MetricCredentials()) > 0 {
		processedStatus.MeterStatuses = context.processUnitMeterStatuses(units)
	}

	versions := make([]status.StatusInfo, 0, len(units))
	for _, unit := range units {
		workloadVersion, err := context.status.FullUnitWorkloadVersion(unit.Name())
		if err != nil {
			processedStatus.Err = apiservererrors.ServerError(err)
			return processedStatus
		}
		versions = append(versions, workloadVersion)
	}
	if len(versions) > 0 {
		sort.Sort(bySinceDescending(versions))
		processedStatus.WorkloadVersion = versions[0].Message
	}
	if processedStatus.WorkloadVersion == "" && context.model.Type() == state.ModelTypeCAAS {
		// We'll punt on using the docker image name.
		caasModel, err := context.model.CAASModel()
...
```